src/components/TextGroup/TextGroup.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Hed from '../Hed/Hed';
+import Kicker from '../Kicker/Kicker';
+import Tagline from '../Tagline/Tagline';
+import styles from './TextGroup.scss';
+
+function TextGroup ( {
+	kicker,
+	tagline,
+	title,
+	type,
+} ) {
+	return (
+		<>
+			{
+				kicker &&
+					<Kicker>
+						<div className={`${styles.kicker} ${styles[ `kicker-${type}` ]}`}>{kicker}</div>
+					</Kicker>
+			}
+			<Hed size="small">{title}</Hed>
+			{
+				tagline &&
+					<Tagline>
+						<div className={styles.tagline}>{tagline}</div>
+					</Tagline>
+			}
+		</>
+	);
+};
+
+TextGroup.propTypes = {
+	/**
+	 * Kicker text. This accepts a string or a node—most notably to accommodate
+	 * bulletins, which have multicolor kickers and sometimes incorporate images.
+	 */
+	kicker: PropTypes.node,
+
+	/**
+	 * Tagline to appear beneath the title.
+	 */
+	tagline: PropTypes.string,
+
+	/**
+	 * Title.
+	 */
+	title: PropTypes.string.isRequired,
+
+	/**
+	 * The type of the text group, which influences color and other formatting.
+	 */
+	type: PropTypes.oneOf( [ 'article', 'default' ] ),
```

Comment:
Do you anticipate there being more available types than these two? If not I think a boolean prop like `isArticle` would be more intuitive

Reply:
I kinda do anticipate more, but not yet sure what they'd be.

Reply:
Yeah, totally. So much of this is subject to change once we start using them more. Will do in next PR